sections/semantics-document-metadata.include

```diff
@@ -392,16 +392,18 @@
   The <dfn element-attr for="link"><code>crossorigin</code></dfn> attribute is a
   <a>CORS settings attribute</a>. It is intended for use with <a>external resource links</a>.
 
-  The types of link indicated (the relationships) are given by the value of the
-  <dfn element-attr for="link"><code>rel</code></dfn> attribute, which, if present, must have a
-  value that is a <a>set of space-separated tokens</a>. The <a>allowed keywords and their
-  meanings</a> are defined in a later section. If the <{link/rel}> attribute is absent, has no
-  keywords, or if none of the keywords used are allowed according to the definitions in this
-  specification, then the element does not create any links.
+  Along with the <{link/href}> attribute, a <{link}> element must have a <{link/rel}> attribute
```

Comment:
"must have" makes the rel attribute required. Suggest this should be "may have" instead?